tests/helpers/blocks.js

```diff
@@ -0,0 +1,125 @@
+import Path from 'path';
+import { readdirSync, readFileSync } from "fs";
+import { props } from '../../scripts/editor';
+
+const pathToBlocksFolder = Path.resolve(__dirname, '..', '..', 'blocks', 'init', 'src', 'Blocks');
+const pathToComponents = Path.resolve(pathToBlocksFolder, 'components');
+const pathToBlocks = Path.resolve(pathToBlocksFolder, 'custom');
+
+const pathToMockBlocksFolder = Path.resolve(__dirname, '..', 'data', 'src', 'Blocks');
+const pathToMockComponents = Path.resolve(pathToMockBlocksFolder, 'components');
+const pathToMockBlocks = Path.resolve(pathToMockBlocksFolder, 'custom');
+
+/**
+ * Returns all manifests from directories inside `dir`.
+ *
+ * @param {string} dir Full path to dir in which we're looking for other 
+ *                     dirs (components / blocks) which have manifest.json inside.
+ *                     Example: 'blocks/init/src/Blocks/components'
+ * @returns {array<object>} Array of manifest.json contents
+ */
+const getAllManifestsFromDir = (dir) => {
+	const manifestDirs = readdirSync(dir);
+	return manifestDirs.map((manifestDir) => {
+		const manifestPath = Path.resolve(dir, manifestDir, 'manifest.json');
+		return JSON.parse(readFileSync(manifestPath));
+	});
+}
+
+/**
+ * Get all components, both the mock ones specifically for tests and the actual libs blocks.
+ * 
+ * @return {array}
+ */
+export const getAllComponentManifests = () => {
+	const componentManifests = getAllManifestsFromDir(pathToComponents);
+	const mockComponentManifests = getAllManifestsFromDir(pathToMockComponents);
+	return [...componentManifests, ...mockComponentManifests];
+}
```

Comment:
How come you didn't write it with

```suggestion
export const getAllComponentManifests = () => [...getAllManifestsFromDir(pathToComponents), ...getAllManifestsFromDir(pathToMockComponents)]
```

Not important at all, and maybe even less readable. But just a suggestion.

Reply:
Generally I like keeping things in variables in case you need to console.log it in the future. But I understand it's a bit worse, will change this in next PR in which I'm fixing tests. 👍